commonlibktx/src/main/java/com/crisolutions/commonlibraryktx/ViewKtx.kt

```diff
@@ -41,6 +42,15 @@ fun View.setVisible(boolean: Boolean) {
     this.isVisible = boolean
 }
 
+fun TextView.setTextOrGone(notEmptyData: String) {
```

Comment:
Could rename `notEmptyData` to just `data`

Reply:
Updated.